examples/fragments/src/main.rs

```diff
@@ -0,0 +1,86 @@
+#[macro_use]
+extern crate yew;
+
+use yew::prelude::*;
+
+type Context = ();
+
+struct Model {
+    counter: usize,
+}
+
+enum Msg {
+    More,
+    Less,
+}
+
+impl Component<Context> for Model {
+    type Msg = Msg;
+    type Properties = ();
+
+    fn create(_: &mut Env<Context, Self>) -> Self {
+        Model {
+            counter: 0,
+        }
+    }
+
+    fn update(&mut self, msg: Self::Msg, _: &mut Env<Context, Self>) -> ShouldRender {
+        match msg {
+            Msg::More => {
+                self.counter = self.counter + 1;
+            }
+            Msg::Less => {
+                if self.counter > 0 {
+                    self.counter = self.counter - 1;
+                }
+            }
+        }
+        true
+    }
+}
+
+impl Renderable<Context, Model> for Model {
+    fn view(&self) -> Html<Context, Self> {
+        html! {
+            <div>
```

Comment:
This would be a good spot for a fragment instead.

Reply:
Ok, it also will be a good test for this feature 👍